kube/services/netpolicy/networkpolicy_aws_es_proxy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: networkpolicy-aws-es-proxy
+spec:
+  podSelector:
+    matchLabels:
+      app: esproxy
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            app: arranger
+      ports:
+         - port: 80
+         - port: 443
+  egress:
```

Comment:
How about we go ahead and public to port `9200` - since that's where everybody expects Elastic Search to be anyway?  And go ahead and get rid of port 443 ...  For sure we need a `targetPort: 9200` ...?

Actually - this is the `networkpolicy` - don't we need an `esproxy-service` too?